src/Carousel/test/CarouselSpec.js

```diff
@@ -91,4 +91,17 @@ describe('Carousel', () => {
 
     ReactTestUtils.Simulate.transitionEnd(instance.querySelector('.rs-carousel-slider'));
   });
+
+  it('Should initialize with the default index position', () => {
+    const instance = getDOMNode(
+      <Carousel defaultActiveIndex={2}>
+        <div>1</div>
+        <div>2</div>
+        <div>3</div>
+        <div>4</div>
+      </Carousel>
+    );
+
+    assert.equal(instance.querySelector('[aria-hidden=false]').textContent, '3');
+  });
```

Comment:
Add a test case for `activeIndex`

Reply:
@simonguo Could you point to a existing test case of a similar scenario? I mean a scenario in which we change the component props externally in the test. 
I've looked through the code and I couldn't find one (even the Panel component doesn't have it) and I am not sure how to implement it using this testing framework.

Reply:
reference test: https://github.com/rsuite/rsuite/blob/main/src/Modal/test/ModalSpec.js#L119-L139

Reply:
Thanks a lot! Test added here https://github.com/rsuite/rsuite/pull/2338/commits/a84e0bfda58ae3a89b127d99ee957e6d70bef966